Translator/Translator/Source/Writer.cpp

```diff
@@ -0,0 +1,16 @@
+#include "Writer.h"
+
+void Writer::writer(vector<Triangle> triangles, vector<double> points) {
+    ofstream outputFile("output.dat");
+
+    for (const auto& triangle : triangles) {
+        outputFile << points[triangle.p1.x] << " " << points[triangle.p1.y] << " " << points[triangle.p1.z] << endl;
+        outputFile << points[triangle.p2.x] << " " << points[triangle.p2.y] << " " << points[triangle.p2.z] << endl;
+        outputFile << points[triangle.p3.x] << " " << points[triangle.p3.y] << " " << points[triangle.p3.z] << endl;
+        outputFile << points[triangle.p1.x] << " " << points[triangle.p1.y] << " " << points[triangle.p1.z] << endl;

```

Comment:
make sure to initialize the data members of point class as private and also provide get function to access them in this class